static/js/matomo-cookiebot.js

```diff
@@ -0,0 +1,29 @@
+var waitForTrackerCount = 0;
+
+function matomoWaitForTracker() {
+  if (typeof _paq === 'undefined' || typeof Cookiebot === 'undefined') {
+    if (waitForTrackerCount < 40) {
+      setTimeout(matomoWaitForTracker, 250);
+      waitForTrackerCount++;
+      return;
+    }
+  } else {
+    _paq.push(['requireConsent']);
+    window.addEventListener('CookiebotOnAccept', function (e) {
+      consentSet();
+    });
+    window.addEventListener('CookiebotOnDecline', function (e) {
```

Comment:
```suggestion
    window.addEventListener('CookiebotOnAccept', function () {
      consentSet();
    });
    window.addEventListener('CookiebotOnDecline', function () {
```

This should work too, right? And as `e` is not used anyway...
Not sure what's best practice in js